stock_available_unreserved/models/product.py

```diff
@@ -15,102 +13,105 @@ class ProductTemplate(models.Model):
     _inherit = "product.template"
 
     qty_available_not_res = fields.Float(
-        string='Quantity On Hand Unreserved', digits=UNIT,
-        compute='_compute_product_available_not_res')
+        string='Quantity On Hand Unreserved',
+        digits=UNIT,
+        compute='_compute_product_available_not_res',
+    )
 
     qty_available_stock_text = fields.Char(
```

Comment:
Can't this field be removed ?
If I understood correctly it just adds ' On Hand' after the value qty_available_not_res, and it could be done in the view instead of having a field for that.